src/docs/operator-guides/migrating-from-outside-sharetribe/index.md

```diff
@@ -28,33 +32,102 @@ Intermediary is an edn (https://github.com/edn-format/edn) data format
 that allows defining data rows for marketplaces. It supports creating
 links between rows via references.
 
+We recommend that you use an .edn library to generate the data and
+validate the .edn syntax to ensure that the file is up to standard.
+
 ### Format
 
 Intermediary is specified as
 [a map](https://clojure.org/reference/data_structures#Maps) with the
 following keys:
 
-- :ident - identifies the target marketplace by marketplace ident,
+- :ident - identifies the target marketplace by marketplace ID,
 - :data - a seq of marketplace data rows. Each row is a 2-tuple of id
   and content.
 
+You can find your marketplace ID in Flex Console > Build > General. Note
+that the anonymised test file needs to specify your test environment
+marketplace ID and your production data file needs to specify your
+production environment marketplace ID.
+
 The id part of row 2-tuple is specified as a tuple of 1 to 3 elements.
 The first element is always an id attribute and identifies the row type.
 The second element can be an alias or an import id. A 3 element version
 has id attr, import id and an alias.
 
+```
+;; 1 element tuple
+[:im.stripeAccount/id]
+
+;; 2 element tuple with import id
+[:im.image/id #uuid "58afd8e1-e336-4ca4-a1e7-ff1d91856a6c"]
+
+;; 2 element tuple with alias
+[:im.user/id :user/jane]
+
+;; 3 element tuple
+[:im.listing/id #uuid "b074e697-ab0c-4746-a195-c58d73606b1f" :listing/rock-sauna]
+```
+
 Import ids can be given to rows and they should be unique for the
-marketplace and they are of type UUID. Aliases can be used to reference
-rows from other rows. Aliases are useful when the data is written by
-hand. Import ids are useful for programmatically generated / exported
-data. However, note that import ids are only used in the import phase
-and they can not be mapped to existing resource ids. Content part of row
-2-tuple is a map that is processed by tuple expansion function
+marketplace and have type [UUID](#uuid). Aliases can be used to
+reference rows from other rows. Aliases are useful when the data is
+written by hand. Import ids are useful for programmatically generated /
+exported data.
+
+Note that import ids are only used in the import phase and they can not
+be mapped to existing resource ids. TODO: EXPLAIN BETTER! Content part
+of row 2-tuple is a map that is processed by tuple expansion function
 registered for the content type.
```

Comment:
This sounds like it's in Clojurian, so do we need an "in other words" in English as well or would there be e.g. a reference we could point to? This sentence doesn't really mean anything for a non-Clojure dev and I think most of the people reading this documentation will not speak Clojurian. :)

Reply:
> Note that import ids are only used in the import phase and they can not
be mapped to existing resource ids.

This part is trying to say that intermediary can not be used to update existing resources, even though their ID would match an existing resource. Furthermore, what is defined as the ID of a resource in intermediary, will not be the ID of the resource in Flex. It is merely used in the import.

> Content part of row 2-tuple is a map that is processed by tuple expansion function registered for the content type.

This I suppose we can leave out. It's enough that we've explained that a resource consists of the ID tuple and content map, that's stated somewhere in the doc, right?